Dockerfile

```diff
@@ -20,6 +20,17 @@ RUN wget https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get &&
   ./get && \
   rm get
 
+RUN opsys=linux
+RUN curl -s https://api.github.com/repos/kubernetes-sigs/kustomize/releases/latest |\
+  grep browser_download |\
+  grep $opsys |\
+  cut -d '"' -f 4 |\
+  xargs curl -O -L
+RUN mkdir kustomize
+RUN mv kustomize_*_${opsys}_amd64 kustomize/kustomize
+RUN chmod u+x kustomize/kustomize
```

Comment:
Let's merge the 3 `RUN` statements after the main into the same `RUN`. This will make it obvious that all of the commands in that block are related.
```
RUN opsys=linux curl -s https://api.github.com/repos/kubernetes-sigs/kustomize/releases/latest |\
  grep browser_download |\
  grep $opsys |\
  cut -d '"' -f 4 |\
  xargs curl -O -L && \
  mkdir kustomize && \
  mv kustomize_*_${opsys}_amd64 kustomize/kustomize && \
  chmod u+x kustomize/kustomize
```

Reply:
We should also pin the Kustomize version instead of using `latest` so that our builds are deterministic.

Reply:
Merged and pin to version is 18896544, (https://api.github.com/repos/kubernetes-sigs/kustomize/releases/18896544)